scripts/ci/check-pro-code.ps1

```
@@ -0,0 +1,43 @@
+param(
+    [string]$BaseRef = "origin/main",
+    [string]$HeadRef = "HEAD"
+)
+
+$ErrorActionPreference = "Stop"
+
+$found = $false
+$repoRoot = Resolve-Path (Join-Path $PSScriptRoot "..\..")
+
+function Report-ProHit {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Report-ProHit' uses an unapproved verb.

(PSUseApprovedVerbs)